### ISSUE TITLE
Broken examples

### ISSUE THREAD

**Issue by FabriceSalvaire** [FabriceSalvaire/PySpice#227]
# power-supplies/rectification.py

missing link doc/sphinx/source/examples/power-supplies/rectification.rst

# operational-amplifier/astable.py

`ValueError Unknown argument expression=V(reference, comparator)`

# relay/relay.py

`ERROR - Warning: vclock: no DC value, transient time 0 value used`

# advanced-usages/internal-device-parameters.py

`ERROR - warning, can't find model nmos`

# spice-parser/kicad-example.py

`NameError Invalid plot name`

# switched-power-supplies/buck-converter.py

`ERROR - Warning: vpulse: no DC value, transient time 0 value used`

# passive/capacitor-inductor.py

`Warning: vinput: no DC value, transient time 0 value used`

# diode/diode-recovery-time.py

`ERROR - Warning: vinput: no DC value, transient time 0 value used`

# data-analysis/fft.py

`TypeError object of type <class 'float'> cannot be safely interpreted as an integer.`

# ngspice-shared/ngspice-interpreter.py

`ERROR - Error: no circuit loaded`

# transmission-lines/time-delay.py

`ERROR - Warning: vpulse: no DC value, transient time 0 value used`

# analyses/pole_zero_analysis.py
# analyses/analyses.py

# examples/index.rst

```
Create TOC /PySpice/doc/sphinx/source/examples/index.rst
Traceback (most recent call last):
  File "/opt/python-virtual-env/py38/bin/pyterate", line 11, in <module>
    load_entry_point('Pyterate==2.1.1', 'console_scripts', 'pyterate')()
  File "Pyterate-2.1.1-py3.8.egg/Pyterate/Scripts/pyterate.py", line 102, in main
    rst_factory.process_recursively()
  File "//Pyterate-2.1.1-py3.8.egg/Pyterate/RstFactory/RstFactory.py", line 107, in process_recursively
    '\n'.join([document.path for document in self._document_failures]))
TypeError: sequence item 0: expected str instance, PosixPath found
```

**Comment by FabriceSalvaire** [comment]
# Pyterate

## power-supplies/rectification.py

missing link doc/sphinx/source/examples/power-supplies/rectification.rst

## examples/index.rst

```
Create TOC /PySpice/doc/sphinx/source/examples/index.rst
Traceback (most recent call last):
  File "/opt/python-virtual-env/py38/bin/pyterate", line 11, in <module>
    load_entry_point('Pyterate==2.1.1', 'console_scripts', 'pyterate')()
  File "Pyterate-2.1.1-py3.8.egg/Pyterate/Scripts/pyterate.py", line 102, in main
    rst_factory.process_recursively()
  File "//Pyterate-2.1.1-py3.8.egg/Pyterate/RstFactory/RstFactory.py", line 107, in process_recursively
    '\n'.join([document.path for document in self._document_failures]))
TypeError: sequence item 0: expected str instance, PosixPath found
```

# PySpice

## analyses/pole_zero_analysis.py
**Fixed**
## analyses/analyses.py
**Fixed**

## operational-amplifier/astable.py
**Fixed**

`ValueError Unknown argument expression=V(reference, comparator)`

## spice-parser/kicad-example.py

`NameError Invalid plot name`

## advanced-usages/internal-device-parameters.py

`ERROR - warning, can't find model nmos`

## data-analysis/fft.py

`TypeError object of type <class 'float'> cannot be safely interpreted as an integer.`

## ngspice-shared/ngspice-interpreter.py

`ERROR - Error: no circuit loaded`


## NgSpice

## relay/relay.py

`ERROR - Warning: vclock: no DC value, transient time 0 value used`

## switched-power-supplies/buck-converter.py

`ERROR - Warning: vpulse: no DC value, transient time 0 value used`

## passive/capacitor-inductor.py

`ERROR - Warning: vinput: no DC value, transient time 0 value used`

## diode/diode-recovery-time.py

`ERROR - Warning: vinput: no DC value, transient time 0 value used`

## transmission-lines/time-delay.py

`ERROR - Warning: vpulse: no DC value, transient time 0 value used`

**Comment by FabriceSalvaire** [comment]
```
020-05-27 21:32:38,967 - Pyterate.RstFactory.Document.Document.run - INFO - 
Run documentPySpice/examples/transistor/transistor.py
PySpice/PySpice/Unit/Unit.py:1819: RuntimeWarning: divide by zero encountered in true_divide
  results = super(UnitValues, self).__array_ufunc__(ufunc, method, *args, **kwargs)

2020-05-27 21:32:41,188 - Pyterate.Jupyter.JupyterClient.close - INFO - Stop kernel

```

**Comment by FabriceSalvaire** [comment]
fixed from f5c5e4a
